src/tools/ScenarioMeasurement/Startup.Tests/StartupTests.cs

```diff
@@ -0,0 +1,82 @@
+using ScenarioMeasurement;
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+namespace Startup.Tests
+{
+    public class StartupTests
+    {
+        Logger logger = new Logger("test-startup.log");
+        string traceDirectory = Environment.CurrentDirectory;
+
+
+        [WindowsOnly]
+        public void TestWindowsTraceSession()
+        {
+            string sessionName = "test-windows-session";
+            string traceName = "test-windows-trace";
+            var session = new WindowsTraceSession(sessionName, traceName, traceDirectory, logger);
+            var parser = new TimeToMainParser();
+            TestSession(session, parser);
+        }
+
+        [LinuxOnly]
+        public void TestLinuxTraceSession()
+        {
+            string sessionName = "test-linux-session";
+            string traceName = "test-linux-trace";
+            var session = new LinuxTraceSession(sessionName, traceName, traceDirectory, logger);
+            var parser = new TimeToMainParser();
+            TestSession(session, parser);
+        }
+
+        [WindowsOnly]
+        public void TestProfileIteration()
+        {
+            string sessionName = "test-profile-iteration-session";
+            string traceName = "test-profile-iteration-trace";
+            var timeToMainParser = new TimeToMainParser();
```

Comment:
looks like this `timeToMainParser` is unused?

Reply:
It's passed in ```ProfileParser(IParser other)``` as a required parameter.

Reply:
oh, duh. sorry. Merge!